keras_hub/src/models/deeplab_v3/deeplab_v3_layers.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2024 The KerasHUB Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import keras
+from keras import ops
+
+
+class SpatialPyramidPooling(keras.layers.Layer):
```

Comment:
please add a test file for `deeplab_v3_layers` and add the [`self.run_layer_tests`](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/tests/test_case.py#L82) for each of them.